src/Entity/ProgrammaticFoundation/Approach.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace AppBundle\Entity\ProgrammaticFoundation;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="programmatic_foundation_approach")
+ */
+class Approach
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\GeneratedValue
+     */
+    private $id;
```

Comment:
could you use `EntityIdentityTrait` instead. It adds `uuid` property too, can be useful for exposing it on URL

Reply:
Do we still keep `id` property ?

Reply:
no need to keep it in the class, as the trait adds `id` and `uuid` properties